src/server/saved_objects/saved_objects_mixin.js

```diff
@@ -0,0 +1,30 @@
+import { SavedObjectsClient } from './client';
+
+import {
+  createCreateRoute,
+  createDeleteRoute,
+  createFindRoute,
+  createReadRoute,
+  createUpdateRoute
+} from './routes';
+
+export function savedObjectsMixin(kbnServer, server) {
+  const prereqs = {
+    getSavedObjectsClient: {
+      assign: 'savedObjectsClient',
+      method(request, reply) {
+        reply(new SavedObjectsClient(
+          server.config().get('kibana.index'),
+          request,
```

Comment:
Huh, I guess I didn't start passing in `callDataCluster` functions until #11114, https://github.com/spalger/kibana/blob/e32833ba2f9c2dc28015f3fac600b744109b7e4f/src/server/index_patterns/mixin.js#L15-L25

Rather than passing in the `request` and `callWithRequest` I just pass in a partially applied version of `callWithRequest` that I called `callDataCluster`.